features/com.google.cloud.tools.eclipse.suite.e45.feature/feature.xml

```diff
@@ -23,5 +23,11 @@
          install-size="0"
          version="94.0.0.qualifier"
          unpack="false"/>
+   <plugin
+       id="com.google.cloud.tools.eclipse.appengine.whitelist"
+       download-size="0"
+       install-size="0"
+       version="94.0.0.qualifier"
```

Comment:
Specify the version as 0.0.0 and it will be filled in automatically at build time